packages/components/src/flex/styles/flex-styles.js

```diff
@@ -60,17 +60,19 @@ const reversedStyles = ( { isReversed } ) => {
 export const Flex = styled.div`
 	box-sizing: border-box;
 	display: flex;
+	width: 100%;
 
 	${ alignStyle };
 	${ justifyStyle };
 	${ gapStyle };
 	${ reversedStyles };
 `;
 
-export const Item = styled.div`
+export const Item = styled.dev`
 	box-sizing: border-box;
 	min-width: 0;
 	max-width: 100%;
+	flex-grow: ${ ( props ) => props.grow };
```

Comment:
I hesitated between a "grow" prop or an "expand" one.

Reply:
I prefer "grow" since it's already CSS/flex terminology.